app/services/idv/flows/in_person_flow.rb

```diff
@@ -32,11 +23,35 @@ class InPersonFlow < Flow::BaseFlow
 
       def initialize(controller, session, name)
         @idv_session = self.class.session_idv(session)
-        super(controller, STEPS, {}, session[name])
+        super(controller, STEPS, ACTIONS, session[name])
       end
 
       def self.session_idv(session)
         session[:idv] ||= { params: {}, step_attempts: { phone: 0 } }
+        # mock out user data for now. once we're actually collecting user data
+        # we can remove this block
+        # WILLFIX: remove this block when we're collecting user data
+        session[:idv][:applicant] ||= {
+          first_name: 'Susan',
+          last_name: 'Smith',
+          middle_name: 'Q',
+          address1: '1 Microsoft Way',
+          address2: 'Apt 3',
+          city: 'Bayside',
+          state: 'NY',
+          zipcode: '11364',
+          dob: '1938-10-06',
+          ssn: '900123123',
+        }
```

Comment:
You may be interested in the work at #6448, since there already exists some mock data which is designed to help us avoid PII leaks. That being said, I understand that this is all temporary code anyways, so not too concerned over it.

```rb
        # mock out user data for now. once we're actually collecting user data
        # we can remove this block
        # WILLFIX: remove this block when we're collecting user data
        session[:idv][:applicant] ||= Idp::Constants::MOCK_IDV_APPLICANT_WITH_SSN
```

Reply:
Nice! I'll pull this in once #6448 is merged

Reply:
Added in f125b23